packages/dara-components/dara/components/graphs/components/edge_encoder.py

```diff
@@ -89,7 +89,8 @@ class VisualEdgeEncoder(StyledComponentInstance):
     ```python
     from cai_causal_graph import CausalGraph
     from cai_causal_graph.type_definitions import EdgeConstraint as EdgeConstraintType
-    from dara.components.graphs import VisualEdgeEncoder, EdgeConstraint
+    from dara.components.graphs import VisualEdgeEncoder
+    from dara.components.graphs.components.edge_encoder import EdgeConstraint
```

Comment:
Did this not work for you? Tried locally and we seem to be able to do as before or `from dara.components import VisualEdgeEncoder, EdgeConstraint`

Reply:
There was feedback that it didn't work (might be due to older version). Tried it and it does work as expected so I added it in the example as well